packages/elements/src/components/ino-input-file/ino-input-file.tsx

```diff
@@ -50,6 +66,57 @@ export class InputFile implements ComponentInterface {
     files: object[];
   }>;
 
+  componentDidLoad(): void {
+    this.configureDragAndDrop();
+  }
+
```

Comment:
I might be wrong ;) but should we also remove the initialized event listeners on `componentDidUnload`?

Reply:
Yes, I think it's better we do so.